### ISSUE TITLE
undocumented limit for `Text`?

### ISSUE THREAD

**Issue by ByteDick** [charmbracelet/huh#501]
found that it can add at most 100 lines in for `Text`. any solution to allow more lines?

**Comment by jonas-grgt** [comment]
`huh.Text` depends on bubbles `textarea.Model` which has a `defaultMaxHeight` of 99

`textarea.Model` exposes a `MaxHeight` field
 

Unfortunately `huh.Text` does not expose its internal `textarea.Model`. 

Maybe it would be a good idea to just expose the `textarea.Model` in `huh.Text` ? 

Or else a delegate func needs to be added for every `textarea.Model` func you want to expose through `huh.Text.

WDYT?

**Comment by meowgorithm** [comment]
Good catch. We removed default character limit in a recent commit (https://github.com/charmbracelet/bubbles/commit/398e92c5ae72b738cd6336a48c85bc21039b8cf8) so this should just be a matter of doing a `bubbles` release and then bumping the dep in `huh`.

**Comment by jonas-grgt** [comment]
@meowgorithm `defaultCharLimit` is indeed set to zero but `defaultMaxHeight` still remains 99 in that commit which is the cause of this issue.

**Comment by meowgorithm** [comment]
True! However we also separated the notions of max lines and max height to allow for large inputs. Here's the full PR:

https://github.com/charmbracelet/bubbles/pull/672

Let me know if I'm wrong, but I _believe_ this should solve for this one.

**Comment by meowgorithm** [comment]
Agreed. Thanks for helping get to the bottom of this one (and thanks @ByteDick for flagging).

**Comment by bashbunni** [comment]
Hey, I'm going to close this as Bubbles `v0.21.0` is released and Huh now depends on it on `main`. Thanks for bringing this to our attention!
